src/validators/execution.py

```diff
@@ -106,23 +107,29 @@ async def get_latest_network_validator_public_keys() -> Set[HexStr]:
     return new_public_keys
 
 
-async def get_withdrawable_assets() -> tuple[Wei, HexStr | None]:
+async def get_harvest_params() -> HarvestParams | None:
```

Comment:
Naming: `get_harvest_params` and `fetch_harvest_params`.
Almost the same.
I need more distinctive naming.
For example: rename `get_harvest_params` to `fetch_harvest_params_if_can_harvest`